src/components/menu/menu-trigger/menu-trigger.directive.ts

```diff
@@ -33,7 +34,12 @@ export class MenuTriggerDirective implements OnInit, OnDestroy {
     private _portal: TemplatePortal;
 
     /** Store the reference to the overlay */
-    private _overlayRef: OverlayRef;
+    private _overlayRef?: OverlayRef;
+
+    /** Get the aria controls for accessibility */
+    get ariaControls(): string | null {
+        return this.menu?.isMenuOpen ? this.menu?.id + '-menu' : null;
```

Comment:
I think you should store this inner id in a separate property rather than duplicating this magic string in two places.